system/lib/fetch/asmfs.cpp

```diff
@@ -1054,7 +1054,7 @@ static long open(const char* pathname, int flags, int mode) {
   return (long)desc;
 }
 
-long __syscall_open(long path, long flags, ...)
```

Comment:
These changes to use `at()` variants increase code size slightly. Very little, but still, is it worth the benefits here? Those aren't clear to me.

Reply:
Yes, indeed.  I think its worth it for the reduce complexity and maintenance. 

It also a code size win for programs that call both flavors of these functions and fewer JS imports.

Reply:
I agree it reduces complexity somewhat. I also agree it's a code size win for programs that use both. But I worry that maybe the common case is a program that uses just one, and it used to use the simple one, which is the case we'd be regressing. That might be worth measuring at least.

Reply:
Actually I don't think it will be a regression even for simple such programs, at least no more than the single line which is the call to `SYSCALLS.calculateAt`.      `SYSCALLS.calculateAt` is part of the `SYSCALLS` objects which IIRC clusure is not able to remove anyway.. so I think the `calculateAt` method gets included in any case.   But I can/will measure.

Reply:
Given that we just landed af4a408d6 is seems a reasonable extension to land this too.